octoprint_v8theme/static/js/v8theme.js

```diff
@@ -1,7 +1,157 @@
 $(function() {
     function V8ThemeViewModel(parameters) {
         var self = this;
-        self.temp = parameters[0]
+        self.temperature = parameters[0];
```

Comment:
You should use 2 spaces for indentation.  I know you didn't start the file, but that's what basically all other JavaScript is.